examples/templates/community-templates.md

```diff
@@ -14,4 +14,5 @@ maintained, please submit a pull request to improve this list. Thank you!
 - [m.lan/coder-templates](https://gitlab.com/m.lan/coder-templates) - Kubernetes template with DinD.
 - [jsjoeio/coder-templates](https://github.com/jsjoeio/coder-templates) - Docker templates that prompt for dotfiles and base Docker image.
 - [8Bitz0/coder-rust-template @ GitLab](https://gitlab.com/8Bitz0/coder-rust-template) - Coder templates with various Linux distros for out-of-the-box Rust development.
-- [sharkymark/v2-templates](https://github.com/sharkymark/v2-templates) - Kubernetes, Docker, AWS, Google Cloud, Azure templates, videos, emoji links, and API examples
+- [sharkymark/v2-templates](https://github.com/sharkymark/v2-templates) - Kubernetes, Docker, AWS, Google Cloud, Azure templates, videos, emoji links, and API examples.
+- [Update Coder Template](https://github.com/marketplace/actions/update-coder-template) - A github action to automate coder template changes.
```

Comment:
Could we put this under a new title? Like `## GitHub Actions`. It feels quite distinct from the templates themselves.

Reply:
hmm ok.

Reply:
IMO we can put it in this same file. Perhaps we also could change `## Content` => `## Templates` so it’s more natural to have a second (sub)title.